### ISSUE TITLE
Argument ordering

### ISSUE THREAD

**Issue by WoLfulus** [rliebz/tusk#53]
**Describe the bug**
I'm having issues with arguments... seems like its not being sorted/passed properly. 

If I run several times a command for example `command arg1 arg2` and `arg1` accepts a list "x, y, z", it will eventually throw me an exception saying
`Value "bla" for argument arg1 must be one of [x, y, z]`

**To Reproduce**
Just use args with a list of values, run sevenral times

**Expected behavior**
Always have the same order passed to the task

**Environment**
 - OS: Windows
 - Version (`tusk --version`): 0.4.6

**Comment by rliebz** [comment]
Thanks for opening an issue!

I'm having trouble reproducing this—it may be because I'm misunderstanding the issue.

Here's an example `tusk.yml` based on your description:

```yaml
tasks:
  command:
    args:
      arg1:
        values: [a, b, c]
      arg2:
        values: [d, e, f]
```

And from my local testing, correct usage will always produce this output:
```
$ tusk command a d
command $ echo a d
a d
```

And passing incorrect arguments will always produce an error:
```
$ tusk command d x
Error value "d" for argument arg1 must be one of [a b c]
```

Can you provide a minimal example of a `tusk.yml` and the arguments you're passing when you see this?

**Comment by WoLfulus** [comment]
Config

```
options:
  namespace:
    type: string
    usage: The images namespace user/org (defaults to directus).
    environment: DIRECTUS_IMAGE_NAMESPACE
    default: directus
  prefix:
    type: string
    usage: The image prefix to use
    environment: DIRECTUS_IMAGE_PREFIX
    default: ""
  push:
    type: bool
    usage: Pushes the containers to the registry if specified.
    default: false

tasks:
  base:
    usage: Builds a base image
    description: |
      Builds a base image and pushes it to the registry if specified
    args:
      kind:
        type: string
        usage: The base image kind (apache, nginx, ...)
        default: apache
        values:
          - apache
      version:
        type: string
        usage: The tag name of the image
        default: latest
    run:
      - command: "echo \"Unknown base kind: base/${kind}\""

```

Logs (no changes between runs)

```
D:\Projects\project\docker
λ tusk base apache b
base $ echo "Unknown base kind: base/apache"
Unknown base kind: base/apache

D:\Projects\project\docker
λ tusk base apache b
base $ echo "Unknown base kind: base/apache"
Unknown base kind: base/apache

D:\Projects\project\docker
λ tusk base apache b
base $ echo "Unknown base kind: base/apache"
Unknown base kind: base/apache

D:\Projects\project\docker
λ tusk base apache b
base $ echo "Unknown base kind: base/apache"
Unknown base kind: base/apache

D:\Projects\project\docker
λ tusk base apache b
Error value "b" for argument kind must be one of [apache]
```

**Comment by rliebz** [comment]
Thanks for the info. I've managed to reproduce this locally and will hopefully have a fix out soon.

**Comment by WoLfulus** [comment]
Awesome, thank you

**Comment by rliebz** [comment]
Fixed in [v0.4.7](https://github.com/rliebz/tusk/releases/tag/v0.4.7).

**Comment by WoLfulus** [comment]
That was fast! Thank you!
